install.sh

```diff
@@ -72,20 +72,18 @@ done
 echo "📁 Creating directory '$dir_name'"
 mkdir -p "$dir_name" && cd "$dir_name" || { echo "❌ Failed to create/access directory '$dir_name'"; exit 1; }
 
-# Copy the twenty/packages/twenty-docker/docker-compose.yml file in it
+# Fetch docker-compose.yml from the branch corresponding to the version or main
 echo -e "\t• Copying docker-compose.yml"
-curl -sLo docker-compose.yml https://raw.githubusercontent.com/twentyhq/twenty/$branch/packages/twenty-docker/docker-compose.yml
+curl -sLo docker-compose.yml https://raw.githubusercontent.com/twentyhq/twenty/$version/packages/twenty-docker/docker-compose.yml
 
-# Copy twenty/packages/twenty-docker/.env.example to .env
+# Set REDIS_URL or REDIS_HOST depending on the version
 echo -e "\t• Setting up .env file"
-curl -sLo .env https://raw.githubusercontent.com/twentyhq/twenty/$branch/packages/twenty-docker/.env.example
+curl -sLo .env https://raw.githubusercontent.com/twentyhq/twenty/$version/packages/twenty-docker/.env.example
 
 # Replace TAG=latest by TAG=<latest_release or version input>
 if [[ $(uname) == "Darwin" ]]; then
-  # Running on macOS
```

Comment:
keep the comments, they are useful here, why remove them?

Reply:
Ok I will fix it